ocs_ci/ocs/resources/pvc.py

```diff
@@ -120,3 +123,33 @@ def get_all_pvcs(namespace=None):
     )
     out = ocp_pvc_obj.get()
     return out
+
+
+def verify_pv_exists_in_backend(
```

Comment:
@Akarsha-rai I think it would be better if this function will be a class method of PVC class. This way, we'll be able to add the option to verify the PV existence as part of PVC create() method.

Reply:
changes will be done in next commit